dev/reference/sql/constraints.md

```diff
@@ -12,11 +12,11 @@ TiDB supports the same basic constraints supported in MySQL with the following e
 
 - `PRIMARY KEY` and `UNIQUE` constraints are checked lazily by default. By batching checks until when the transaction commits, TiDB is able to reduce network communication. This behavior can be changed by setting `tidb_constraint_check_in_place` to `TRUE`.
 
-- `FOREIGN KEY` constraints are not currently enforced by DML.
+- TiDB supports creating foreign key constraints, but these constraints are ineffective in DML statements.
 
 ## Foreign Key
 
-TiDB currently only supports `FOREIGN KEY` creation in DDL commands. For example:
+TiDB currently supports `FOREIGN KEY` constraints. For example:
```

Comment:
I think "TiDB currently supports `FOREIGN KEY` creation in DDL commands.`" would be better, since we don't fully support it now.

Reply:
@wjhuang2016 Comments addressed. PTAL again, thanks.
Also I'll update the Chinese version in a new PR.